File: Logical-tier-spring/demo/src/model/Account.java
package model;

import java.io.Serializable;
public class Account implements Serializable

{

    private String username,password, role;

    public Account(String username, String password, String role)
    {
        this.username = username;
        this.password = password;
        this.role = role;
    }

    public String getUsername()
    {
        return username;
    }

    public String getPassword()
    {
        return username;
    }

    public String getRole()
    {
        return role;
    }

    @Override public String toString()
    {
        return "User{" + "username='" + username + '\'' + ", password='" + password
                + '\'' + ", role='" + role + '\'' + '}';
    }
}

File: BlazorSep3/model/Account.cs
namespace BlazorSep3.model
{
    public class Account
    {
        public string username { get; set; }
        public string password { get; set; }

        public string Token { get; set; }

        public string Role { get; set; }

        public override string ToString()
        {
            return $@"Username: {username} Password: {password} has been returned from spring";
        }
    }
}